main/fastcgi.c

```diff
@@ -1469,6 +1474,7 @@ int fcgi_accept_request(fcgi_request *req)
 			return -1;
 		}
 		if (fcgi_read_request(req)) {
+			req->hook.on_read();
```

Comment:
> Also a more general question: Can someone tell me why the poll/select is only done for the fd<0 case? Why is this not applicable to the keepalive case?
Erm, maybe because in keepalive case the connection is already there and no need to add it to select() set?